app/index.html

```diff
@@ -68,9 +68,9 @@ <h5 class="card-title" id="WelcomeMessage">Please sign-in to see your profile an
     <script type="text/javascript" src="./graphConfig.js"></script>
     <script type="text/javascript" src="./ui.js"></script>
     
-    <!-- <script type="text/javascript" src="./authRedirect.js"></script>   -->
+    <script type="text/javascript" src="./authRedirect.js"></script>  
     <!-- uncomment the above line and comment the line below if you would like to use the redirect flow -->
```

Comment:
If the lines commented out are changed, should this comment also be changed?

Reply:
Sorry I will revert this.